.travis.yml

```diff
@@ -82,8 +82,8 @@ matrix:
       - ABI=64
       - GAP=master
       - PACKAGES=required
-      - THRESHOLD=98
-
+      - THRESHOLD=94
```

Comment:
This is... unfortunate. Is there a deep reason why you can't get better code coverage in `isomorph.gi` via `isomorph.tst`, or is it just that you've not got round to it?

If you're not going to get the coverage up, I'd prefer you to put this back up to `98` anyway, and instead put `isomorph.tst` into `.covignore`.

Reply:
I'm not sure what is going on here, but @flsmith agreed to try to remove this change yesterday, if I remember correctly.

Reply:
Caused by my confusion about nauty and now fixed